app/grandchallenge/notifications/templates/actstream/follow_form.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load profiles %}
+{% load humanize %}
+
+{% block title %}Subscriptions - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item active" aria-current="page">Subscriptions</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+    <form method="post">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input type="submit" value="Update">
+    </form>

```

Comment:
We should use django crispy forms, but this is linked to my comments about the FollowCreate view.